src/main/java/foxtrot/uniform/charlie/kilo/service/implementation/AuthenticationServiceImpl.java

```diff
@@ -32,6 +36,9 @@ public User register(String email, String password) {
             user.setEmail(email);
             user.setSalt(HashUtil.getSalt());
             user.setPassword(HashUtil.hashPassword(password, user.getSalt()));
+            ShoppingCart shoppingCart = new ShoppingCart();
+            shoppingCart.setUser(user);
+            shoppingCartDao.add(shoppingCart);
             return userDao.add(user);
```

Comment:
please see checklist - what should be done first - create user or create cart for user?

Reply:
Checklist asks me to think about it, probably it's asking too much.